modules/maven/galasa-maven-plugin/pom.xml

```diff
@@ -189,8 +189,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>11</source>
-					<target>11</target>
+					<source>17</source>

```

Comment:
I was getting a bad error on class loading where the Java 11 code format wasn't working here. Changing it up to Java 17 solved my problem. Not sure if this should be submitted or not. Depends if others are finding the same thing when they build everything from scratch.

Reply:
Makes sense. I'm not sure either on the delivery of this though. Will it break users with older Java runtimes?

Reply:
Personally I've not had any issues building everything from scratch but can't say for others.

Reply:
Reverted it now I've upgraded to mvn 3.9.0